### ISSUE TITLE
Runtimewarning in docs

### ISSUE THREAD

**Issue by mhvk** [mhvk/baseband#139]
Testing currently gives:
```
docs/baseband/tutorials/getting_started.rst
  /usr/lib/python3/dist-packages/astropy/units/quantity.py:639: RuntimeWarning: invalid value encountered in true_divide
    result = super().__array_ufunc__(function, method, *arrays, **kwargs)

-- Docs: http://doc.pytest.org/en/latest/warnings.html
```
It would be good to see if we can get rid of it (not sure where it happens)

**Comment by mhvk** [comment]
After discussion: the problem arises because we create a header in the streamwriter using `fromvalues`, where the argument includes a `time`, but not a `samplerate`. Probably need to ensure that (a) `StreamWriter` passes on `sample_rate` and `fromvalues` uses it also for EDVs that do not have that.

**Comment by mhvk** [comment]
fixed by #147
